WebDriverAgentTests/IntegrationTests/XCElementSnapshotHelperTests.m

```diff
@@ -63,35 +63,35 @@ - (void)testParentMatchingType
   XCTAssertEqual(windowSnapshot.elementType, XCUIElementTypeWindow);
 }
 
-- (void)testParentMatchingOneOfTypes
+- (void)testFindVisibleParentMatchingOneOfTypes
 {
   [self goToAttributesPage];
   XCUIElement *todayPickerWheel = self.testedApplication.pickerWheels[@"Today"];
   XCTAssertTrue(todayPickerWheel.exists);
   [todayPickerWheel resolve];
-  XCElementSnapshot *datePicker = [todayPickerWheel.lastSnapshot fb_parentMatchingOneOfTypes:@[@(XCUIElementTypeDatePicker), @(XCUIElementTypeWindow)]];
```

Comment:
Please do not remove tests for `fb_parentMatchingOneOfTypes` and if possible add test that covers your issue but is not covered by `fb_parentMatchingOneOfTypes`.